packages/ckeditor5-basic-styles/src/bold/boldediting.ts

```diff
@@ -60,6 +60,8 @@ export default class BoldEditing extends Plugin {
 							styles: [ 'font-weight' ]
 						};
 					}
+
+					return null;
```

Comment:
@arkflpc maybe we should allow `undefined` for `MatcherPattern` return value?

Reply:
I'd keep `null`